WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/ShippingLabelFormViewModel.swift

```diff
@@ -84,7 +85,8 @@ final class ShippingLabelFormViewModel {
                 if let predefinedPackage = option.predefinedPackages.first(where: { $0.id == package.packageID }) {
                     let boxID = predefinedPackage.id
                     let customsForm = customsForms.first(where: { $0.packageID == boxID })
-                    return ShippingLabelPackageSelected(boxID: boxID,
+                    return ShippingLabelPackageSelected(id: UUID().uuidString,
```

Comment:
Similar question as above - should we use initializer with predefined package here?

Reply:
:D same response as above